src/web/mjs/connectors/NovelCool.mjs

```diff
@@ -0,0 +1,75 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Mangafest extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'novelcool';
+        super.label = 'Novel Cool';
+        this.tags = [ 'spanish', 'manga', 'webtoon'];
+        this.url = 'https://es.novelcool.com';
+    }
+
+    async _getMangas() {
+        let morePages = true;
+        let page = 1;
+
+        let mangas = [];
+        while (morePages) {
+            let request = new Request(new URL(`/category/index_${page}.html`, this.url), this.requestOptions);
+            let data = await fetch(request);
+
+            if ( data.url == `https://es.novelcool.com/category/index_${page++}.html` ) {
+                data = this.createDOM(await data.text());
+                data = [...data.querySelectorAll('div.book-list div.book-info a')];
+
+                mangas.push(...data.map(manga => {
+                    return {
+                        id: this.getRootRelativeOrAbsoluteLink(manga, this.url),
+                        title: manga.title.trim()
+                    };
+                }));
+
+            } else {
+                morePages = false;
+            }
+        }
+
+        return mangas;
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.chapter-item-list a');
+        return data.map(chapter => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(chapter, this.url),
+                title: chapter.title.trim()
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        let request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        let page_links = await this.fetchDOM(request, 'select.sl-page');
+        page_links = [...page_links[0].querySelectorAll('option')].map(link => link.value);
+
+        let pages = [];
+        for( const link of page_links) {
```

Comment:
`_getPages` should not scrape to all pages, consider using `this.createConnectorURI(...)` such as in [LELScan.mjs](https://github.com/manga-download/hakuneko/blob/4d0af6153b291747f862b9833b2ce09e6c91ed40/src/web/mjs/connectors/LELScan.mjs#L82), but with async/await instead of promises